adoc/deployment-ecp.adoc

```diff
@@ -88,3 +102,43 @@ image::deploy-loadbalancer-ip.png[]
 . Switch the filter dropdown to `Instance Name` and enter the string you specified for `stack_name` in the `terraform.tfvars` file.
 . Find the Floating IPs on each of the nodes of your cluster.
 ====
+
+=== Log into the cluster notes
+
+. Conneting into the cluster nodes can be accomplished only via SSH key-based authentication thanks to the ssh-public key injection done earlier via {tf}. You can log in as `sles` user.
++
+----
+# With ssh-agent running in the background
+ssh sles@<node-ip-address>
+
+# Without ssh-agent running
+ssh sles@<node-ip-address> -i <path-to-your-ssh-public-key>
```

Comment:
Slightly incorrect: the `-i` parameter should point to the `private` file: for reference from the man page:

```man
-i identity_file
             Selects a file from which the identity *(private key)* for public key authentication is read.  The default is ~/.ssh/id_dsa, ~/.ssh/id_ecdsa, ~/.ssh/id_ed25519 and
             ~/.ssh/id_rsa.  Identity files may also be specified on a per-host basis in the configuration file.  It is possible to have multiple -i options (and multiple identities
             specified in configuration files).  If no certificates have been explicitly specified by the CertificateFile directive, ssh will also try to load certificate information
             from the filename obtained by appending -cert.pub to identity filenames.
```

Reply:
True. Thanks!